lib/src/css_mirroring/rulesets_processor.dart

```diff
@@ -50,9 +51,18 @@ RemovalResult editFlippedRuleSet(MirroredEntity<RuleSet> mirroredRuleSet,
   } else {
     /// Add direction attribute to RuleId for direction-specific RuleSet.
     var dir = enumName(flippedDirection);
-    mirroredRuleSet.flipped.prepend(trans, ':host-context([dir="$dir"]) ');
+    prepend(mirroredRuleSet.flipped, trans, ':host-context([dir="$dir"]) ');
 
     subTransaction.commit();
     return RemovalResult.removedSome;
   }
 }
+
+void prepend(Entity<RuleSet> e, BufferedTransaction trans, String s) {
```

Comment:
prependToEachSelector (otherwise feels like we're prepending once to the entity)